src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CodeFix.Testing/CodeFixTest`1.cs

```diff
@@ -692,7 +692,11 @@ private async Task VerifyProjectAsync(ProjectState newState, Project project, IV
 
             try
             {
-                if (expectedNumberOfIterations >= 0)
+                if (expectedNumberOfIterations == 0 && numberOfIterations <= 0)
```

Comment:
💭 We don't even need to check `numberOfIterations` right?

```suggestion
                if (expectedNumberOfIterations == 0)
```